src/containerapp/azext_containerapp/custom.py

```diff
@@ -2197,3 +2200,26 @@ def show_env_managed_identity(cmd, name, resource_group_name):
         r["identity"] = {}
         r["identity"]["type"] = "None"
         return r["identity"]
+
+def set_environment_telemetry(cmd,
+                                 name,
+                                 resource_group_name,
+                                 app_insights_connection_string = None,
+                                 open_telemetry_traces_destinations = None,
+                                 open_telemetry_logs_destinations = None,
+                                 open_telemetry_metrics_destinations = None,
+                                 open_telemetry_dataDog_site = None,
+                                 open_telemetry_dataDog_key = None):
```

Comment:
PEP 8: E127 continuation line over-indented for visual indent

Reply:
`open_telemetry_dataDog_site `and `open_telemetry_dataDog_key`: Argument name should be lowercase